REFERENCE.md

```diff
@@ -93,6 +94,40 @@ Data type: `Any`
 
 The data for when there is no cache yet
 
+### `extlib::cidr_to_range`
+
+Type: Ruby 4.x API
+
+Imported by Tim 'bastelfreak' Meusel into voxpupuli/extlib because Yelp/netstdlib got abandoned
+
+#### Examples
+
+##### calling the function
+
+```puppet
+extlib::cidr_to_range('127.0.0.1/8')
+```
+
+#### `extlib::cidr_to_range(Variant[Stdlib::IP::Address::V4::CIDR,Stdlib::IP::Address::V6::CIDR] $ip)`
+
+Imported by Tim 'bastelfreak' Meusel into voxpupuli/extlib because Yelp/netstdlib got abandoned
+
+Returns: `Variant[Array[Stdlib::IP::Address::V4::Nosubnet],Array[Stdlib::IP::Address::V6::Nosubnet]]` IPv6 or IPv4 ip range without net/broadcast
+
+##### Examples
+
+###### calling the function
+
+```puppet
+extlib::cidr_to_range('127.0.0.1/8')
```

Comment:
I sincerely hope no one calls this with a `/8`...

Reply:
not sure that's much worse than the standard `/64` IPv6 address range…

Reply:
Well, IPv4 /8 is 2**24 while IPv6 /64 is 2**64. Every IPv4 address is 32 bits. 32 bits * 2**24 = 67 MB (assuming optimal storage, which it isn't). For IPv6 it's 128 bits * 2**64 = 256 EiB. If my mat is correct, you really don't want to do this for IPv6.